src/core/canonical_constructor.jl

```diff
@@ -28,7 +28,7 @@ end
 function _canonical_model_init(bus_numbers::Vector{Int64},
                               optimizer::Union{Nothing,JuMP.OptimizerFactory},
                               transmission::Type{S},
-                              time_range::UnitRange{Int64}; kwargs...) where {S <: PM.AbstractPowerFormulation}
```

Comment:
Lookahead has a specific connotation that I don’t think aligns with this usage. Unless we are using it to describe the time periods that overlap with future sequential solves, I think we should rename. If you don’t like time range, then perhaps time periods or temporal range...

Reply:
Ok, I am trying to distinguish the number of steps measured in discrete values from ones that use time stamps. How about `time_steps`? or `step_counts`?

Reply:
I like `time_steps`